book/translation_note.asc

```diff
@@ -0,0 +1,13 @@
+=== Anmerkung der Übersetzer ===
+
+Die vorliegende Übersetzung des englischen Original-Textes wurde ausschließlich von freiwilligen, nicht professionellen Übersetzern geleistet.
+
+Wir bitten um Nachsicht, falls manche Formulierungen nicht immer sehr elegant sind.
```

Comment:
```suggestion
Wir bitten um Nachsicht, falls die eine oder andere Passage nicht in einwandfreies Hochdeutsch übersetzt wurde.
```

Reply:
```suggestion
Wir bitten um Nachsicht, falls die eine oder andere Passage nicht in korrektem, flüssigem Deutsch übersetzt wurde.
```
Das „Hochdeutsch“ möchte ich vermeiden. Ich wollte mit dem Satz mehr darauf hinaus, dass benachbarte Sätze hier öfter Wort-Wiederholungen enthalten. Bei gut formulierten Texten wird man das seltener finden. Für mich liest sich ein solcher Text dann wesentlich flüssiger.
Mit meinem „korrekt“ bin ich aber auch nicht ganz zufrieden – wäre „flüssig“ allein die bessere Alternative?

Reply:
Ok, jetzt verstehe ich, um was es dir geht. Ich bin immer noch nicht glücklich - wenn man hier nur liest 'nicht in flüssiges Deutsch', entsteht für mich der Eindruck, dass es eine schlechte Übersetzung ist. Damit verkaufen wir uns unter Wert, finde ich. Vielleicht ist dein ursprünliches 'elegant' doch die beste Variante. 'Deutsch' würde ich dann gar nicht erwähnen. Vorschlag:
```suggestion
Wir bitten um Nachsicht, falls die eine oder andere Passage nicht elegant ausformuliert wurde.
```
oder
```suggestion
Wir bitten um Nachsicht, falls die eine oder andere Passage nicht elegant übersetzt wurde.
```